pynestml/codegeneration/resources_nest/ModuleClass.jinja2

```diff
@@ -65,8 +65,13 @@
 #include "{{moduleName}}.h"
 
 {% for neuron in neurons %}
+    {%- if neuron.is_compartmental_model -%}
```

Comment:
Templates to their own directory according to the new scheme (https://github.com/nest/nestml/pull/669).